.gitignore

```diff
@@ -18,6 +18,7 @@ profile
 DerivedData
 *.hmap
 *.ipa
+Package.resolved
```

Comment:
I guess we don't check in the package.resolved because this is a library, not a final target ?

Reply:
it can be both but thinking about it again it seems to be better to check it in